dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/SourceCodeRepoInterface.java

```diff
@@ -176,8 +176,9 @@ public void readFile(String repositoryId, Version<?> tag, Collection<SourceFile>
 
     /**
      * Checks to see if a particular source code repository is properly setup for issues like token scope
+     * @param deepCheck do a deeper check that repo is set up
```

Comment:
deepCheck seems to be more checking that the token is valid, or more valid? rather than something about a specific repo